test/jdk/java/security/KeyStore/PKCS12/java.security

```diff
@@ -0,0 +1,2 @@
+# override keystore.type property and make it not exist
```

Comment:
nit: maybe the comment is easier to understand if changed to "set keystore.type property to nothing, so default value will be used".